reports/aspace2alma/get_ao2MARC_data.rb

```diff
@@ -10,6 +10,12 @@ def remove_tags(text)
   text.to_s.gsub(%r{</?[\D\S]+?>}, '')
 end
 
+#literal ampersands are sprinkled throughout; they break the XML
+#check 6 characters out to make sure it's not already an html entity, and if not, entify it
+def entify_ampersands(text)
```

Comment:
This seems like a great method for testing! Can we pair on adding a test or two for it?